Sources/MapboxNavigation/NavigationCameraStateTransition.swift

```diff
@@ -172,26 +173,45 @@ public class NavigationCameraStateTransition: CameraStateTransition {
         }
         
         // Check whether the pitch change is larger than a certain threshold when current camera state is following.
-        let updateCameraPitch = (state == .following) ? (abs(mapView.cameraState.pitch - pitch) >= 1.0) : true
+        let updateCameraPitch = (state == .following) ? (abs(mapView.cameraState.pitch - pitch) >= CGFloat(pitchThreshold)) : true
         
         if updateCameraPitch {
-            let pitchTimingParameters = UICubicTimingParameters(controlPoint1: CGPoint(x: 0.0, y: 0.0), controlPoint2: CGPoint(x: 1.0, y: 1.0))
-            animatorPitch = mapView.camera.makeAnimator(duration: duration, timingParameters: pitchTimingParameters) { (transition) in
+            animatorPitch = mapView.camera.makeAnimator(duration: duration, timingParameters: timingParameters) { (transition) in
                 transition.pitch.toValue = pitch
-                transition.anchor.toValue = anchor
-                transition.padding.toValue = padding
             }
             
             animatorPitch?.startAnimation()
         }
+        
+        if let animatorAnchor = animatorAnchor, animatorAnchor.isRunning {
+            animatorAnchor.stopAnimation()
+        }
+        
+        animatorAnchor = mapView.camera.makeAnimator(duration: duration, timingParameters: timingParameters) { (transition) in
```

Comment:
We should make this animator conditional on whether the value has actually changed, to avoid extraneous animations that could impact performance. It’s probably less likely to occur for the anchor than for the other parameters, but you never know if the application could trigger extra relayouts somehow.